reactor-aeron-benchmarks/src/main/resources/latency-report.json

```diff
@@ -0,0 +1,22 @@
+{
+  "layout": {
+    "title": "Latency Aeron vs Reactor-Aeron vs Reactor-Netty  (C5.xlarge)",
```

Comment:
let's change this title - we will reuse this template for other tests

Reply:
there is a reuse this is a specific configuration for this project 

for reuse its possible to use this one
https://github.com/scalecube/trace-reporter/blob/develop/src/main/resources/chart_template.json